plugins/dependencies/src/main/resources/refreshVersions-rules/testing-version-alias-rules.txt

```diff
@@ -10,8 +10,8 @@ org.junit.jupiter:junit-jupiter(-*)
 io.kotest:*
    ^^^^^^
 
-io.kotest.extensions:kotest-extensions-*
-                     ^^^^^^.^^^^^^^^^^.^
+io.kotest.extensions:*
+   ^^^^^^.^^^^^^^^^^
```

Comment:
@jmfayard Are you sure about that??

https://search.maven.org/search?q=g:io.kotest.extensions

Reply:
@LouisCAD I think so, all dependencies inside `io.kotest.extension` that are maintained have the same version, 1.2.5
Other kotest dependencies are in another namespace

Reply:
Let's ask @sksamuel if he agrees with the fact that everything non version 1.2.5 (so everything but the arrow extension) has been discontinued.

<img width="662" alt="Screenshot 2022-08-17 at 18 56 08" src="https://user-images.githubusercontent.com/6975237/185198417-22a5cf7a-9687-4933-b1e5-c95e7c3f6412.png">

Reply:
Hi,

The 1.0.1 konform is current. The arbs is current. The pitest is current.

Basically, some external modules moved to io.kotest.extensions instead of io.kotest and had their versions reset, around 18 months ago.

If you have a central list I can check them over.